### PR TITLE
Prepare for test set evaluation of tiny GIN for WNV on PCBA AID 588689 by enabling test set evals and setting default hyperparameters to those found in search on validation set

### PR DESCRIPTION
**Pull request by willy-b**
Prepare for test set evaluation by turning on reporting of test set performance at the end of the run (after training with selection of best model epoch by validation set performance).

Also, update indentation to be consistent in `main_gin.py`.

Relevant validation set checks for PCBA AID 588689 before moving on to test set eval next:

- Tiny GIN (32K parameters; 64d hidden dimension): https://colab.research.google.com/drive/17JILSAkHj4IvKIpksBpjD0yrSyIVGZTv (public link)

- Really Tiny GIN (little bit worse but very very small <2K parameters; 8d hidden dimension): https://colab.research.google.com/drive/1cwPXmJLkonOpW3aw6Iyjk_2vEXPfKLPG (public link)

Regression check on this branch (training/validation data only): 
https://colab.research.google.com/drive/185Oof8RYU56eZ8CFHwRZ_uyXIQ5Wnjnf (request access if desired)

### CODE REVIEW

**Comment by willy-b** [comment]
As I was submitting this my Instagram account generated a suspicious notification as observed from a separate account (despite 2FA; possibly via that mobile device which only has access to my socials, no access to email and not trusted otherwise), sharing content (did not share on IG today, FB only yesterday), clicking notification led to content related to pediatric dentistry, **small delay in running test evaluation to look into that.**

Signed out everywhere and taking appropriate precautions **though could just be IG notification for FB content delayed by a day with a bad deeplink** as see nothing in activity history in account.

instagram_account_or_notifications_somehow_spoofed_temporarily

**Comment by willy-b** [comment]
Results are ready for tiny GIN from this repo (ran on CPU), waiting for OGB baselines to finish, will be a couple of days.

Separate from the model here, I am evaluating the following baselines for comparison on PCBA AID 588689:

- OGB Team GIN baseline (1,885,506 parameters) on PCBA AID 588689 "Flavivirus Genomic Capping Enzyme Inhibition" (test set evaluation)

- OGB Team GIN w/ virtual node baseline (3,336,606 parameters) on PCBA AID 588689 "Flavivirus Genomic Capping Enzyme Inhibition" (test set evaluation)

- OGB Team GCN baseline (528,001 parameters) on PCBA AID 588689 "Flavivirus Genomic Capping Enzyme Inhibition" (test set evaluation) (running on CPU since smaller)

- OGB Team GCN w/ virtual node baseline (1,979,101 parameters) on PCBA AID 588689 "Flavivirus Genomic Capping Enzyme Inhibition" (test set evaluation) (A100)

These are based on the recommended baseline models per OGB guidelines for their new datasets at https://docs.google.com/document/u/0/d/e/2PACX-1vS1hBTYLONRwAU9UxK42USTuRKrt_Yk4H0EhpLvJC_eOrGxbJUtrzDqlIStAFpnwZt2N28B3MuSxgqj/pub?pli=1 which recommend using the baselines from their paper Hu, Weihua, et al. "Open graph benchmark: Datasets for machine learning on graphs." Advances in neural information processing systems 33 (2020): 22118-22133 ( https://arxiv.org/abs/2005.00687 ).

**Comment by willy-b** [comment]
Results are being posted in https://github.com/willy-b/tiny-GIN-for-WNV/pull/16 .

Note that the document cited for OGB team's recommended baselines when evaluating new datasets ( https://docs.google.com/document/u/0/d/e/2PACX-1vS1hBTYLONRwAU9UxK42USTuRKrt_Yk4H0EhpLvJC_eOrGxbJUtrzDqlIStAFpnwZt2N28B3MuSxgqj/pub?pli=1 ) in the previous comment https://github.com/willy-b/tiny-GIN-for-WNV/pull/15#issuecomment-2613404287
is a public document linked from the public ogb.stanford.edu "Dataset overview" page, subsection "Contributing Datasets" (e.g. link as archived by Internet Archive at https://web.archive.org/web/20241220083832/https://ogb.stanford.edu/docs/dataset_overview/ ).
